starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,47 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.2rem;
+    ;
+}
+
+header {
+    background: url('../images/apple-pie.jpg') no-repeat center center;
+    background-size: cover;
+    color: white;
+    font-size: 3rem;
+    text-align: center;
+    padding: 20%;
+}
+
+article {
+    margin: 20px auto;
+    width: 80%;
+}
+
+img {
+    width: 300px;
+    display: block;
+    margin: 20px auto;
+}
```

Comment:
Here you are selecting the html tags directly in order to style them, even though it fills the purpose of the lab,
it's better to give the html class names and style them by their class otherwise here, if you ever add an image/ header / article, they will all have these styles applied to them.